### PR TITLE
jest running

### PR DESCRIPTION
**Pull request by RaeChen07**
run yarn after pulling this brunch will get the test setted up. 
in terminal type "yarn jest" will run all the test.
It will watch all file in __test__ folder and with .test.js(/ts) or .spec.js(/ts) file

### CODE REVIEW

**Review by charliepnnl — COMMENTED** [review]
Can you add a test for `pathToId()` in /src/utilities/paths_utils.tsx?

**Comment by RaeChen07** [comment]
> Can you add a test for `pathToId()` in /src/utilities/paths_utils.tsx?

@charliepnnl do you know where it was used? didn't quite get what this function do.(or a example will be good too)

File: src/utilities/paths_utils.tsx [hunk 1]
```diff
@@ -10,6 +10,12 @@ import { isUndefined, toPath } from 'lodash'
  * @param separator String used to separate the path components (default: "-")
  * @returns 
  */
-export function pathToId(prefix: string = "path", path: string): string {
-  return [prefix, ...toPath(path)].join("-")
+export function pathToId(path: string, prefix?: string, separator?:string): string {
```
Comment (charliepnnl) [inline src/utilities/paths_utils.tsx:13]: This is a good change. Please update the order of the parameters in the documentation to match the argument order.

**Review by charliepnnl — APPROVED** [review]
Approved with one comment that should be addressed.
Please merge this in after addressing the comment.
Then check that the github action works as expected.

**Pull request merged**